spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestPartitionValues.java

```diff
@@ -392,8 +393,7 @@ public void testPartitionedByNestedString() throws Exception {
     // schema and partition spec
     Schema nestedSchema = new Schema(
         Types.NestedField.required(1, "struct",
-            Types.StructType.of(Types.NestedField.required(2, "string", Types.StringType.get()))
-        )
+            Types.StructType.of(Types.NestedField.required(2, "string", Types.StringType.get())))
```

Comment:
Please revert the non-functional changes in this file.

Reply:
Sorry, I miss the code specification, now it has been updated, Could you have a look again ? Thank you.

Reply:
Looks like this is still a problem.

Reply:
Please see the latest code ~